integrations/utils/src/lib.rs

```diff
@@ -55,7 +55,9 @@ pub fn html_parts_separated(
     options: &LeptosOptions,
     meta: Option<&MetaContext>,
 ) -> (String, &'static str) {
-    let pkg_path = &options.site_pkg_dir;
+    let pkg_path = option_env!("CDN_PKG_PATH")
+        .map(String::from)
+        .unwrap_or(format!("/{}", options.site_pkg_dir));
```

Comment:
Two smallish but important things here:
1. `option_env!()` returns `Option<&'static str>`, you can make these a `Cow<'static, str>` and avoid the allocation in the CDN case
2. `.unwrap_or_else(|| format!` is better here because `.unwrap_or` always constructs its argument

So in other words:
```rust
    let pkg_path = option_env!("CDN_PKG_PATH")
        .map(Cow::from)
        .unwrap_or_else(|| format!("/{}", options.site_pkg_dir).into());
```

Reply:
I always forget `Cow`. I stared at my solution at first knowing I missed something haha.
And then I figured since I'm already making a needless allocation, doing another unconditionally wouldn't matter. But you're right, if we can avoid it trivially, why not?